python/tests/voice/test_elevenlabs_pump.py

```
@@ -0,0 +1,469 @@
+        async def recv_audio(self, timeout: float) -> AudioChunk:
+            return AudioChunk(data=b"")
+
+        def is_connected(self) -> bool:
+            return False
+
+    adapter = _Disconnected()
+    adapter.send_audio = AsyncMock()  # type: ignore[method-assign]
+    adapter.recv_audio = AsyncMock(return_value=AudioChunk(data=b""))  # type: ignore[method-assign]
+
+    class _Input:
+        new_messages: list[Any] = []
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)